File: app/src/main/java/ir/food/operatorAndroid/app/EndPoints.java
package ir.food.operatorAndroid.app;


import ir.food.operatorAndroid.BuildConfig;

public class EndPoints {

    /*TODO : check apis and ports before release*/

//    http://192.168.1.145:3000/api/operator/*****
//    http://api.parsian.ir:1881/api/operator/
//    http://192.168.1.145/api/findway/citylatinname/address

    public static final String IP =
            "http://happypizza.ir";
//            "http://192.168.43.93";

    public static final String HAKWEYE_IP =
            "http://happypizza.ir";
//          : "http://192.168.1.145";


    public static final String APIPort = (BuildConfig.DEBUG) ? "3010" : "3010";

    public static final String ACRA_PATH = "http://turbotaxi.ir:6061/api/v1/crashReport";
    public static final String PUSH = "http://turbotaxi.ir:6060";

    public static final String WEBSERVICE_PATH = IP + ":" + APIPort + "/api/operator/v1/";
    public static final String COMPLAINT_PATH = IP + ":" + APIPort + "/api/operator/v1/";
    public static final String ACTIVATE_PATH = IP + ":" + APIPort + "/api/operator/v1/activate";
    public static final String CUSTOMER_PATH = IP + ":" + APIPort + "/api/operator/v1/customer";
    public static final String ORDER_PATH = IP + ":" + APIPort + "/api/operator/v1/order";

    /******************************** Base Api *********************************/

    public static final String APP_INFO = WEBSERVICE_PATH + "app/info";
    public static final String LOG_IN = WEBSERVICE_PATH + "login";
    public static final String SIGN_UP = WEBSERVICE_PATH + "register";
    public static final String LOGIN_VERIFICATION_CODE = WEBSERVICE_PATH + "login/verificationcode";
    public static final String VERIFICATION_CODE = WEBSERVICE_PATH + "verificationcode";
    public static final String ADD_ORDER = ORDER_PATH;
    public static final String CANCEL_ORDER = ORDER_PATH;
    public static final String GET_CUSTOMER = CUSTOMER_PATH;
    public static final String GET_DELIVERY_LOCATION = ORDER_PATH + "/delivery";
    public static final String GET_ORDER = ORDER_PATH;
    public static final String GET_ORDERS = ORDER_PATH;
    public static final String EDIT_ADDRESS = ORDER_PATH+"/editAddress";

    public static final String ENTER_QUEUE = ACTIVATE_PATH;


}

File: app/src/main/java/ir/food/operatorAndroid/app/EndPoints.kt
package ir.food.operatorAndroid.app

object EndPoints {
    /*TODO : check apis and ports before release*/
    const val IP = "http://happypizza.ir"

    const val HAKWEYE_IP = "http://happypizza.ir"

    private const val APIPort = "3010"
    const val ACRA_PATH = "http://turbotaxi.ir:6061/api/v1/crashReport"
    const val PUSH = "http://turbotaxi.ir:6060"
    private const val WEBSERVICE_PATH = "$IP:$APIPort/api/operator/v1/"
    private const val COMPLAINT_PATH = "$IP:$APIPort/api/operator/v1/complaint"
    private val ACTIVATE_PATH = "$IP:$APIPort/api/operator/v1/activate"
    private const val CUSTOMER_PATH = "$IP:$APIPort/api/operator/v1/customer"
    private const val ORDER_PATH = "$IP:$APIPort/api/operator/v1/order"
    private const val SUPPORT = "$IP:$APIPort/api/operator/v1/order/support"

    /******************************** Base Api  *********************************/

    const val APP_INFO = WEBSERVICE_PATH + "app/info"
    const val LOG_IN = WEBSERVICE_PATH + "login"
    const val SIGN_UP = WEBSERVICE_PATH + "register"
    const val LOGIN_VERIFICATION_CODE = WEBSERVICE_PATH + "login/verificationcode"
    const val VERIFICATION_CODE = WEBSERVICE_PATH + "verificationcode"

    /******************************** register order Api  *********************************/

    const val ENTER_QUEUE = "${WEBSERVICE_PATH}queue/enter"
    const val EXIT_QUEUE = "${WEBSERVICE_PATH}queue/exit"
    const val GET_CUSTOMER = CUSTOMER_PATH
    const val GET_PRODUCTS = "$ORDER_PATH/product"
    const val ADD_ORDER = ORDER_PATH
    const val EDIT_ORDER = "$ORDER_PATH/editOrder"
    const val SEND_MENU = "$ORDER_PATH/menu"
    const val CALCULATE_BILL = "${ORDER_PATH}/bill"

    /******************************** support order Api  *********************************/

    const val GET_ORDERS_LIST = SUPPORT
    const val GET_ORDER_DETAILS = "$ORDER_PATH/v1/"
    const val EDIT_ADDRESS = "$ORDER_PATH/editAddress"
    const val ADD_COMPLAINT = COMPLAINT_PATH
    const val CANCEL_ORDER = ORDER_PATH
    const val GET_DELIVERY_LOCATION = "$ORDER_PATH/delivery/"
    const val ARCHIVE_ADDRESS = "$ORDER_PATH/cust/archive/address"

}